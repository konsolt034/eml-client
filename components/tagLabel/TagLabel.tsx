"use client";

import styled from "styled-components";

const DisplayTag = styled.span`
  border-radius: 5px;
  font-size: 12px;
  padding: 2px 4px;
  background-color: #6c6c6c;
  color: #f9f9f9;
`;

export default DisplayTag;
