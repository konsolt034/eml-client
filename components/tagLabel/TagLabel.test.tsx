import { render } from "@testing-library/react";

import TagLabel from ".";

describe("Component - TagLabel", () => {
  it("should renders the component", () => {
    const { container } = render(<TagLabel>Horror</TagLabel>);
    expect(container).toMatchSnapshot();
  });

  it("should render text", () => {
    const { getByText } = render(<TagLabel>Documentary</TagLabel>);
    expect(getByText("Documentary")).toBeTruthy;
  });
});
