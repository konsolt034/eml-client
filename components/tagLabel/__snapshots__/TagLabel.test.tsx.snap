// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - TagLabel should renders the component 1`] = `
<div>
  <span
    class="sc-bcPKhP djlObm"
  >
    Horror
  </span>
</div>
`;
